### PR TITLE
added a travis config file for automated test

### PR DESCRIPTION
**Pull request by GarthDB**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,10 +2,11 @@
   "name": "phonegap-docs",
   "version": "0.0.1",
   "description": "The official PhoneGap documentation.",
+  "repository": "git@github.com:phonegap/phonegap-docs.git",
   "dependencies": {
     "autoprefixer-stylus": "^0.5.0",
-    "docpad": "6.73.5",
-    "docpad-plugin-browsersync": "./vendor/docpad-plugin-browsersync",
+    "docpad": "^6.78.1",
+    "docpad-plugin-browsersync": "^2.0.6",
```
Comment (mwbrooks) [inline package.json]: Should we not be vendoring this plugin instead of using the published version that is not compatible with node `0.12.x`?

Comment (GarthDB) [inline package.json]: Yeah, for sure. I had it the other way locally.

Comment (mwbrooks) [inline package.json]: Hmm... but now it's merged and we are no longer using the vendor copy of `docpad-plugin-browsersync`....

**Comment by mwbrooks** [comment]
We should also add a README.md badge for Travis CI

File: package.json (hunk 1 above)
Comment (GarthDB) [inline package.json]: I reverted the file. I'll add the badge shortly.

**Pull request merged**